client/web/web.go

```diff
@@ -270,8 +270,9 @@ func (s *Server) serveLoginAPI(w http.ResponseWriter, r *http.Request) {
 }
 
 type authResponse struct {
-	OK      bool   `json:"ok"`                // true when user has valid auth session
-	AuthURL string `json:"authUrl,omitempty"` // filled when user has control auth action to take
+	OK       bool   `json:"ok"`                 // true when user has valid auth session
+	AuthURL  string `json:"authUrl,omitempty"`  // filled when user has control auth action to take
+	SynoAuth bool   `json:"synoAuth,omitempty"` // true when synology auth is needed
```

Comment:
Thoughts on making this a `Type` field instead? With string "syno"/"tailscale"/...

We take the synology action if the `Type` is "syno" and `OK` is false.

Reply:
done.  Switched an new `authType` enum that has "synology" and "tailscale" values.